Update com.redhat.component label

julienlim opened the pull request:
The value of com.redhat.component label should be the same as the name of the Brew package for the container being built.

@aweiteka Please review and approve.

julienlim committed:
Update com.redhat.component label

The value of com.redhat.component label should be the same as the name of the Brew package for the container being built.

TomasTomecek commented:
Those two are identical. Also Brew is our internal service, not sure whether it's a good idea to reference it in public document.

ssampat commented:
@TomasTomecek do you have a suggestion on how it can be reworded more appropriately? Right now, this requirement is not mentioned anywhere and it's not clear what the value of the com.redhat.component label should be.

TomasTomecek commented:
How about

    | com.redhat.component | Respective component name in [Red Hat Bugzilla](https://bugzilla.redhat.com/), so users can easily file bugs against this image. |

And for our internal docs, we can even mention that this is the package name inside internal koji instance.

julienlim commented:
@TomasTomecek I'm not sure it's clear enough to the user as it assumes that the image maintainer/owner knows that the BZ component name == Build system package name.   Could we instead say it's the Build system package name?

@ssampat Thoughts?

TomasTomecek commented:
@julienlim 

How about then:

    | com.redhat.component | Respective component name in [Red Hat Bugzilla](https://bugzilla.redhat.com/) and a package name field inside internal build system. |

julienlim committed:
Updated the description for com.redhat.component

Updated the description for com.redhat.component per recent GitHub conversation.

julienlim commented:
@TomasTomecek I've updated the description again per your suggestion.  Please review.

Thanks!

ssampat commented:
Looks good to me!

TomasTomecek commented:
+1, I can't merge though :/

rhatdan commented:
LGTM
